packages/components/src/components/hds/time/index.hbs

```diff
@@ -0,0 +1,59 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+{{! IMPORTANT: we need to add "squishies" here (~) because otherwise the whitespace added by Ember causes extra space around the time element - See https://handlebarsjs.com/guide/expressions.html#whitespace-control }}
+{{~#let this.display as |display|~}}
+  {{~#if this.isValidDate~}}
+    {{~#if this.hasTooltip~}}
+      <Hds::TooltipButton
+        class="hds-time-wrapper"
+        @text={{if
+          display.options.tooltipFormat
+          (hds-format-date this.date display.options.tooltipFormat)
+          this.isoUtcString
+        }}
+        @placement="bottom"
+        @extraTippyOptions={{hash showOnCreate=this.isOpen}}
+      >
+        <Hds::Time::Single
+          @date={{this.date}}
+          @isoUtcStringInner={{this.isoUtcString}}
+          @displayInner={{this.display}}
```

Comment:
suggestion: unless I'm missing something, I believe it's safe to name these `isoUtcString` and `display` – we have getters with the same name, so I don't see a reason to add the `Inner` suffix. Of course, this change should cascade in signature and single subcomponent
```suggestion
          @isoUtcString={{this.isoUtcString}}
          @display={{this.display}}
```

Reply:
I renamed display args.